dotnet/test/common/PrintTest.cs

```diff
@@ -89,5 +90,38 @@ public void MarginsCannotBeNull()
         {
             Assert.That(() => new PrintOptions { PageMargins = null }, Throws.InstanceOf<ArgumentNullException>());
         }
+
+        [Test]
+        public void PageNumberCannotHaveInvalidValues()
+        {
+            Assert.That(() => new PrintOptions().AddPageToPrint(-1), Throws.TypeOf<ArgumentOutOfRangeException>());
+            Assert.That(() => new PrintOptions().AddPageRangeToPrint(null), Throws.TypeOf<ArgumentException>());
+            Assert.That(() => new PrintOptions().AddPageRangeToPrint(""), Throws.TypeOf<ArgumentException>());
+            Assert.That(() => new PrintOptions().AddPageRangeToPrint("1-2-3"), Throws.TypeOf<ArgumentException>());
+            Assert.That(() =>
+            {
+                var options = new PrintOptions();
+                options.AddPageToPrint(1);
+                options.AddPageToPrint(1);
+            }, Throws.TypeOf<ArgumentException>());
+        }
+
+        [Test]
+        public void PageSizeCannotHaveNegativeValues()
+        {
+            Assert.That(() => new PrintOptions.PageSize { Height = -1 }, Throws.TypeOf<ArgumentOutOfRangeException>());
+            Assert.That(() => new PrintOptions.PageSize { HeightInInches = -1 }, Throws.TypeOf<ArgumentOutOfRangeException>());
+            Assert.That(() => new PrintOptions.PageSize { Width = -1 }, Throws.TypeOf<ArgumentOutOfRangeException>());
+            Assert.That(() => new PrintOptions.PageSize { WidthInInches = -1 }, Throws.TypeOf<ArgumentOutOfRangeException>());
+        }
```

Comment:
Two things here that are potentially bugs:

I tried to add this assertion:
```csharp
Assert.That(() =>
{
    var options = new PrintOptions();
    options.AddPageRangeToPrint("1-2");
    options.AddPageToPrint(1);
}, Throws.TypeOf<ArgumentException>());
```

but no exception is thrown. Validating against this is not trivial, but there's already so much validation on this method that I figured it may be worthwhile.

Second, the following code does nothing:
```csharp
options.AddPageRangeToPrint("1");
```

No exception is thrown, the single page is not added, nothing. I can fix that simply. Should it throw, or accept the single page?

Reply:
Opened https://github.com/SeleniumHQ/selenium/issues/14783